src/SwedbankPay/Api/Client.php

```diff
@@ -0,0 +1,425 @@
+<?php
+// phpcs:ignoreFile
+
+namespace SwedbankPay\Api;
+
+use SwedbankPay\Api\Client\Exception;
+use SwedbankPay\Api\Response;
+
+/**
+ * Class Client
+ * @note For backward compatibility for previous versions
+ * @deprecated Use SwedbankPay\Api\Client\Client instead of
+ * @package SwedbankPay\Api
+ * @SuppressWarnings(PHPMD.CamelCasePropertyName)
+ */
+class Client extends \SwedbankPay\Api\Client\Client
+{
+    const USER_AGENT = 'SwedbankPay HTTP Client';
+    const VERSION = '2.0.1';
```

Comment:
Can the version number be read from `composer.json` instead of being duplicated here?